src/app/components/job-list-item/job-list-item.component.scss

```diff
@@ -1,27 +1,48 @@
 @import '../../styles/colors';
 
 .adress-field {
-  color: $color-gray-400;
+  padding-top: 2px;
+  padding-bottom: 2px;
+  font-size: 14px;
 }
 
 .job-list-container {
+  color: $color-gray-400;
+  max-width: 840px;
+  margin: 0 auto;
+  cursor: pointer;
+  cursor: hand;
   padding: 1em;
-  border-bottom: 1px solid $color-gray-100;
+  border-bottom: 1px solid $color-gray-50;
   p {
     margin-bottom: 2px;
   }
 }
 
+.date-container {
+  color: $color-gray-700;
+  text-transform: uppercase;
+  text-align: center;
+  font-size: 18px;
+  font-weight: 500;
+  line-height: 20px;
+  top: 0.85em;
+  float: left;
+  width: 100px;
+}
+.job-meta-details {
+  margin-left: 120px;
+}
+
 .job-list-container-header {
-  font-weight: bold;
+  color: $color-gray-700;
+  font-size: 18px;
```

Comment:
Change to named font size? Otherwise we risk having _many_ different sizes all over the the place :)